dockerfile/Dockerfile

```diff
@@ -0,0 +1,48 @@
+FROM fedora:rawhide
+
+# Prepare environment and install build dependencies
+RUN dnf update -y &&\
+  dnf install -y \
+  'dnf-command(copr)' \
+  'dnf-command(builddep)' \
+  curl \
+  /usr/bin/xargs \
+  rpm-build \
+  python3-pip && \
+  dnf copr enable -y @rhinstaller/Anaconda && \
+  dnf copr enable -y @storage/blivet-daily
+
+RUN curl -L https://raw.githubusercontent.com/rhinstaller/anaconda/master/anaconda.spec.in -o /tmp/anaconda.spec && \
+  sed -i 's/@PACKAGE_VERSION@/0/' /tmp/anaconda.spec && \
+  sed -i 's/@PACKAGE_RELEASE@/0/' /tmp/anaconda.spec && \
+  sed -i 's/%{__python3}/python3/' /tmp/anaconda.spec && \
+  dnf builddep -y /tmp/anaconda.spec && \
+  rpmspec -q --requires /tmp/anaconda.spec | grep -v anaconda | xargs -d '\n' dnf install -y && \
+  dnf clean all
```

Comment:
This does not help, and is just counter-productive here. The next RUN step again calls dnf, so this wipes the cache for that, and doesn't clean up after. Please merge two dnf steps into one `RUN` layer, and run `dnf clean all` as the last step.

Reply:
My bad, I missed that the next RUN is also dnf. Fixed.